### DISCUSSION TITLE
MobaXterm

### DISCUSSION THREAD

**Discussion by notAutomne** [catppuccin/catppuccin#2180] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

Since recent major version update [MobaXterm](https://mobaxterm.mobatek.net/) (not the sexiest, but the greatest shell) finally support importing theme. 

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Closing as discussion should continue in https://github.com/catppuccin/catppuccin/issues/2197
